crates/re_viewer/src/ui/rerun_menu.rs

```diff
@@ -385,9 +385,16 @@ fn debug_menu_options_ui(
     ).on_hover_text("Show a debug overlay that renders the picking layer information using the `debug_overlay.wgsl` shader.");
 
     re_ui.checkbox(ui,
-        &mut app_options.show_blueprint_in_timeline,
-        "Show Blueprint in the Time Panel",
-    ).on_hover_text("Show the Blueprint data in the Time Panel tree view. This is useful for debugging the internal blueprint state.");
+        &mut app_options.show_blueprint_timeline,
+        "Show a timeline of the blueprint data.",
+    )
```

Comment:
nit:

This would make better as a radio button pair, as it is not an additive option, but a switch (what do we show in the timeline - recording or blueprint?)

something like

``` rust
ui.horizontal(|ui| {
    ui.label("Timeline:"):
    ui.radio_value(&mut show_blueprint_timeline, false, "Recording");
    ui.radio_value(&mut show_blueprint_timeline, true, "Blueprint");
});